### ISSUE TITLE
[Bug] Opening Wino from tray reloads some mail accounts

### ISSUE THREAD

**Issue by pohuing** [bkaankose/Wino-Mail#421]
**Describe the bug**
My @t-online.de email account reloads if I open Wino from tray. After a second or two it'll discard all emails in the email listing and refetch from scratch.

**Expected behavior**
It should just show the emails as it did before closing to tray.

**Screenshots**
https://github.com/user-attachments/assets/ffbdedc7-c867-4558-93d1-82f1847ca650

**Additional context**
That mail account did have an issue on the Win10 Mail App where its mail list would turn empty if I search. The account works fine in all other mail clients I have testes(K9, Thunderbird, Gnome Mail, Macos' Apple Mail).

**Comment by Tiktack** [comment]
Am I understand this correctly, when you open wino you see emails adding? 
If yes - then everything is fine. At the left, below account name you can see progress bar, which means this account still doing initial sync. As soon it completes, you should not see such behavior.

**Comment by pohuing** [comment]
This happens repeatedly. It happened just now when I clicked the mail notification to open the github email to this thread...
None of the emails that are visually being added are new.

**Comment by Tiktack** [comment]
From recording I can see that it's IMAP, it gmail never faced such behavior, but still, could you leave wino open for awhile until initial sync is done and try again?

**Comment by pohuing** [comment]
How do I see that the initial sync is done, the little toast at the bottom? The sync progress bar does finish and disappear. Same for the small green toast.

**Comment by Tiktack** [comment]
If progress bar dissappeared - init sync is done or was terminated ( if you closed Wino from tray). But it's very weird what you see, when open a folder, it queries from DB some number of items in one query, in should list them all together immediately.
We need @bkaankose help.

**Comment by pohuing** [comment]
Tell me if I'm wrong but I think at the end of this video the entire account is properly synced. If I close to tray and reopen it after this clip it'll show the same loading behaviour again.

https://github.com/user-attachments/assets/daa23418-6fe8-4928-9c74-ad2ba0166bca

**Comment by Tiktack** [comment]
Yes, it looks like a bug. It syncs everything again for some reason. It should not....

**Comment by mikrofisch** [comment]
Just wanted to confirm that this issue still exists and I would consider it a blocker for using Wino-Mail with IMAP accounts. There are other open bugs related to sync as well, perhaps related.

**Comment by Vinsup** [comment]
> Just wanted to confirm that this issue still exists and I would consider it a blocker for using Wino-Mail with IMAP accounts. There are other open bugs related to sync as well, perhaps related.

I have to admit it is the same for me. It is quite an annoying issue.

**Comment by Vinsup** [comment]
Calendar would be very nice for sure, but I could rely on Notion Calendar for the time being to ma'age my Google Calendar in my case.
Thanks for the follow up.

**Comment by bkaankose** [comment]
New Beta release v 1.10.0 has imap improvements included. 

https://github.com/bkaankose/Wino-Mail/releases/tag/v1.10.1

Please give this version a try and let me know if the new synchronizers fixed your problem.

**Comment by pohuing** [comment]
Unfortunately this did not change anything.

**Comment by bkaankose** [comment]
New version v2.0.13.0 is released to Microsoft Store yesterday. Closing as fixed. If this is still reproducible, please reopen with steps.
